.github/workflows/ci.yml

```diff
@@ -311,6 +346,8 @@ jobs:
         env:
           RUSTFLAGS: --cfg tokio_unstable -Dwarnings
 
+  # TODO: Once https://github.com/cross-rs/cross/issues/716 lands,
+  # switch to using cargo-nextest with cross to speedup tests
```

Comment:
FWIW, I guess my [setup-cross-toolchain-action](https://github.com/taiki-e/setup-cross-toolchain-action) (currently used in no-atomic-u64 job) should already work with nextest.

Reply:
I've also changed job `cross-check` to use setup-cross-toolchain-action.